i18n/en/building.md

```diff
@@ -162,7 +163,7 @@ $ git clone --recursive https://github.com/monero-project/kovri
 
 ### 2. Build kovri and submodule dependencies with one command
 ```bash
-$ make  # to decrease build-time, run `make -j <available CPU cores>`. Example: `make -j4`
+$ make release-static # to decrease build-time, run `make -j <available CPU cores>`. Example: `make -j4`
```

Comment:
>Dynamic builds do not link Boost libraries correctly, making
`release-static` builds necessary.

If there is a problem with the build system, then open an issue in the meta or kovri repo. Also, I can't reproduce.

The only real reason this line in the docs would be needed is because the current installer will only install via innosetup - which is only called during release-static.

Reply:
I couldn't build without `release-static`. Will dig deeper, and make sure it's not just something on my build setup.

If it ends up being a legitimate build problem, I'll open an issue on meta or kovri repo.